startmicro

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+regex="([^\/]+)-([^-]+)-cache-config.yaml"
+node_yaml='(() => {
+var yaml = require("js-yaml");
+var fs   = require("fs");
+var doc  = yaml.safeLoad(fs.readFileSync(process.argv[1], "utf8"));
```

Comment:
right now we supply these as commandline arguments.  Will this break that?  For instance, if the doc name is omitted?